--- a/dist/configure	2022-12-26 13:46:24.000000000 -0800
+++ b/dist/configure	2022-12-27 11:35:26.000000000 -0800
@@ -18756,6 +18756,7 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -18792,7 +18793,8 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
-main() {
+#include <stdlib.h>
+int main() {
 	pthread_cond_t cond;
 	pthread_mutex_t mutex;
 	pthread_condattr_t condattr;
@@ -18828,6 +18830,7 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -18864,7 +18867,7 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
-main() {
+int main() {
 	pthread_cond_t cond;
 	pthread_mutex_t mutex;
 	pthread_condattr_t condattr;
@@ -18899,6 +18902,7 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -18933,7 +18937,8 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
-main() {
+#include <stdlib.h>
+int main() {
 	pthread_cond_t cond;
 	pthread_mutex_t mutex;
 	pthread_condattr_t condattr;
@@ -18967,6 +18972,7 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -19001,7 +19007,8 @@
 /* end confdefs.h.  */
 
 #include <pthread.h>
-main() {
+#include <stdlib.h>
+int main() {
 	pthread_cond_t cond;
 	pthread_mutex_t mutex;
 	pthread_condattr_t condattr;
@@ -19038,6 +19045,7 @@
 /* end confdefs.h.  */
 
 	#include <synch.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -19069,6 +19077,7 @@
 
 	#include <thread.h>
 	#include <synch.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -19099,6 +19108,7 @@
 
 	#include <thread.h>
 	#include <synch.h>
+#include <stdlib.h>
 int
 main ()
 {
@@ -20743,7 +20753,7 @@
 /* end confdefs.h.  */
 
 #include <sys/time.h>
-main() {
+int main() {
 	struct timespec t;
 	return (clock_gettime(CLOCK_MONOTONIC, &t) != 0);
 }
@@ -21634,7 +21644,7 @@
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
-		main() {
+		int main() {
 			$db_cv_seq_type l;
 			unsigned $db_cv_seq_type u;
 			char buf[100];
