diff -c /Users/john/Development/Gramps-Build/gramps-40/src/libgexiv2-0.5.0/Makefile\~ /Users/john/Development/Gramps-Build/gramps-40/src/libgexiv2-0.5.0/Makefile
*** a/Makefile	Mon Oct  1 12:19:33 2012
--- b/Makefile	Thu Mar  7 10:46:43 2013
***************
*** 150,155 ****
--- 150,156 ----
  		$(EXPANDED_INSTALLED_HEADER_FILES)      \
  		$(EXPANDED_SRC_FILES)                   \
  		--library $(LIBRARY_BIN)                \
+ 		--libtool=libtool                       \
  		--pkg=gobject-2.0
  
  clean:

Diff finished.  Thu Mar  7 10:47:48 2013
