*** a/_imagingft.c	Sat Oct 31 17:44:12 2009
--- b/_imagingft.c	Thu May 22 12:30:02 2014
***************
*** 70,76 ****
--- 70,80 ----
      const char* message;
  } ft_errors[] =
  
+ #if defined(USE_FREETYPE_2_1)
+ #include <freetype2/fterrors.h>
+ #else
  #include <freetype/fterrors.h>
+ #endif
  
  /* -------------------------------------------------------------------- */
  /* font objects */
--- a/setup.py	Sun Nov 15 08:06:10 2009
+++ b/setup.py	Fri May 23 11:26:20 2014
@@ -348,7 +348,7 @@
                     dir = os.path.join(root, "Tk.framework", "Headers")
                     add_directory(self.compiler.include_dirs, dir, 1)
                     break
-            if frameworks:
+            if False:
                 exts.append(Extension(
                     "_imagingtk", ["_imagingtk.c", "Tk/tkImaging.c"],
                     extra_compile_args=frameworks, extra_link_args=frameworks
