diff -c /Users/john/Development/gtk-sources/Pillow-2.7.0/setup.py\~ /Users/john/Development/gtk-sources/Pillow-2.7.0/setup.py
--- a/setup.py	Fri Jan  2 03:52:21 2015
+++ b/setup.py	Sun Mar 22 14:54:09 2015
@@ -575,7 +575,7 @@
             exts.append(Extension(
                 "PIL._webp", ["_webp.c"], libraries=libs, define_macros=defs))
 
-        if sys.platform == "darwin":
+        if _tkinter and sys.platform == "darwin":
             # locate Tcl/Tk frameworks
             frameworks = []
             framework_roots = [

Diff finished.  Sun Mar 22 20:37:52 2015
