From: Ross Gammon <rossgammon@mail.dk>
Date: Fri, 24 Jan 2014 23:20:58 +0100
Subject: Correct resource path in setup.py

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 3f702ae..9622d75 100644
--- a/setup.py
+++ b/setup.py
@@ -248,7 +248,7 @@ class install(_install):
                                      'utils', 'resource-path')
         with io.open(resource_file, 'w', encoding='utf-8',
                      errors='strict') as fp:
-            path = os.path.abspath(os.path.join(self.install_data, 'share'))
+            path = os.path.join('/usr', 'share')
             if sys.version_info[0] < 3:
                 path = unicode(path)
             fp.write(path)
